tools/packaging/kata-deploy/kata-cleanup/base/kata-cleanup.yaml

```diff
@@ -13,7 +13,7 @@ spec:
         labels:
           name: kubelet-kata-cleanup
     spec:
-      serviceAccountName: kata-label-node
+      serviceAccountName: kata-deploy
```

Comment:
suggestion: Suffix this with "sa" or "service-account" as the daemonset is named similarly as well which is not an issue but just makes things more explicit and avoids confusion while describing either or.

Reply:
I will change, thanks!